lightningd/log.c

```diff
@@ -53,7 +53,7 @@ static void log_to_file(const char *prefix,
 			iso8601_s, prefix, str, dir, hex);
 		tal_free(hex);
 	} else 	if (!continued) {
-		fprintf(logf, "%s %s %s\n", iso8601_s, prefix, str);
+		fprintf(logf, "%s %s %s%s\n", iso8601_s, prefix, level == LOG_BROKEN ? "**BROKEN** " : "", str);
```

Comment:
I'd prefer actually just prefixing all log lines with their respective log level rather than making this special case for `BROKEN`.

Reply:
I concur.  For `LOG_BROKEN` we can special case it with highlighting asterisks.